packages/core/src/Stores/client-store.js

```diff
@@ -451,10 +451,23 @@ export default class ClientStore extends BaseStore {
         );
 
         reaction(
-            () => [this.is_logged_in, this.is_authorize, this.is_passkey_supported],
+            () => [
+                this.is_logged_in,
+                this.is_authorize,
+                this.is_passkey_supported,
+                this.root_store.ui?.is_real_acc_signup_on,
+                this.root_store.ui?.is_from_signup_account,
+                this.root_store.traders_hub.is_eu_user,
+            ],
             () => {
                 if (this.is_logged_in && this.is_authorize && this.is_passkey_supported) {
-                    this.fetchShouldShowEffortlessLoginModal();
+                    if (this.root_store.ui?.is_from_signup_account) {
```

Comment:
This will delay the call for passkeys'list API request. isnt.
do we need to delay the API call as well